Sources/FBLPromises/include/FBLPromise.h

```diff
@@ -55,14 +55,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)reject:(NSError *)error NS_REFINED_FOR_SWIFT;
 
-+ (instancetype)new NS_UNAVAILABLE;
-- (instancetype)init NS_UNAVAILABLE;
-
 /**
  Add an object to the set of pending objects to keep strongly while the promise is pending.
  */
-- (void)addToPendingObjects:(id)object NS_REFINED_FOR_SWIFT;
+- (void)__addPendingObject:(id)object;
```

Comment:
Let's move it under an extension just to keep it separate from the rest of public APIs

```
@interface FBLPromise<Value>()

/**
 Adds an object to the set of pending objects to keep strongly while the promise is pending.
 Used by the Swift wrappers to keep them alive until the underlying ObjC promise is resolved.

 @param object An object to add.
 */
- (void)addPendingObject:(id)object NS_REFINED_FOR_SWIFT;

@end
```